From b9942be4f9643917d6e7176ac7685b9100cf77d4 Mon Sep 17 00:00:00 2001
From: Vitaly Andrianov <vitalya@ti.com>
Date: Wed, 24 Jan 2018 12:33:36 -0500
Subject: [PATCH 156/156] arm: dts: am57xx: add jailhouse dts files into
 Makefile

There are am572x-*-jailhouse.dts files added to the tree awhile ago, but
missed in the makefile. This commit fixes this issue.

Signed-off-by: Vitaly Andrianov <vitalya@ti.com>
---
 arch/arm/boot/dts/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 228f6da..a053986 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -593,6 +593,8 @@ dtb-$(CONFIG_SOC_DRA7XX) += \
 	am57xx-cl-som-am57x.dtb \
 	am57xx-sbc-am57x.dtb \
 	am572x-idk.dtb \
+	am572x-idk-jailhouse.dtb \
+	am572x-evm-jailhouse.dtb \
 	am572x-idk-lcd-osd101t2045.dtb \
 	am572x-idk-lcd-osd101t2587.dtb \
 	am574x-idk.dtb \
-- 
1.9.1

